### DISCUSSION TITLE
Binary file differences on 10GB+ files in a cloned repo using lfs

### DISCUSSION THREAD

**Discussion by zvirtual1** [git-lfs/git-lfs#5005] [Q&A]
Hello,

I'm new to lfs, but have a weird problem that is plaguing us from using git lfs with confidence. We have a repo where we need to save some binaries that are up to 10gb in size, these files are .gdb tables. When I make a bare repo (git version 2.36.1.windows.1), install lfs (3.1.4), no tracking is enabled, the add, commit the binaries. Most  of the commits seem to binary compare, but when cloning the repo and comparing using the Beyond Compare tool, we are seeing differences on a few of these files (.gdb, .loz, .las), in fact they don't even open after cloning. We are trying to find out why select these files would not be binary comparisons. 

1. This is a local repo, no remote exists. At this time, the repo and the clone is on the same physical machine for testing.
2. lfs file tracking is not enabled as we don't care abut size since everything is local. We just want the ability to track changes to the files. 
3. git-config = 
[core]
	repositoryformatversion = 0
	filemode = false
	bare = true
	symlinks = false
	ignorecase = true
[lfs]
	repositoryformatversion = 0

Any help would be greatly appreciated.

**Comment by bk2204** [comment]
Hey,

If you haven't used `git lfs track` or otherwise modified `.gitattributes` to set up Git LFS, then Git LFS isn't being used at all, and you're simply storing files in Git.  It may be that there's a problem on Windows with very large files which isn't a problem on Unix, but I thought that was recently fixed.  You will absolutely have to be using a 64-bit Git binary for this to work, though.

If you can confirm that you don't have a `.gitattributes` file, then you're probably going to want to ask the Git for Windows folks [on their issue tracker](https://github.com/git-for-windows/git/).  You might also ask on the [Git mailing list](https://git-scm.com/community).  If you do have a `.gitattributes` file, it would be great if you could share it so we can see what's going on.

If you're not using Git LFS, my guess as to what's going on here is that you may have some sort of line-ending conversion enabled and that these files are getting converted, which is a common problem on Windows.  If that's the case, you'd definitely want to add something like `*.gdb -text` (and respectively for the other types) to `.gitattributes` and then run `git add --renormalize .` in the original repo, and see if that fixes your problem.

**Reply by zvirtual1** [reply]
Brain, thank you for the reply! You are correct in your assumption that we do not have a .gitattributes file as we installed git lfs, but did not track any files . I'll move forward with the Git for Windows folks, thanks for pointing me in the right direction.

**Reply by PhilipOakley** [reply]
See also [Teach Git to handle huge files in smudge/clean #3487](https://github.com/git-for-windows/git/pull/3487#) and the [Git list discussions](https://lore.kernel.org/git/pull.1068.git.1635320952.gitgitgadget@gmail.com/t/#m3990d4473e7028579656f5992c0692d20eee1659). We're not fully there yet.
